hw/top_earlgrey/ip/ast/doc/_index.md

```diff
@@ -0,0 +1,1006 @@
+<td>I</td>
+<td></td>
+<td></td>
+<td>Ground</td>
+</tr>
+<tr class="odd">
+<td>AGND</td>
+<td>I</td>
+<td></td>
+<td></td>
+<td>Analog ground (see AVCC for further details)</td>
+</tr>
+<tr class="even">
+<td><strong>Power Control and Reset</strong></td>
+<td></td>
+<td></td>
+<td></td>
+<td></td>
+</tr>
+<tr class="odd">
+<td>flash_power_down_h_o</td>
+<td>O</td>
+<td>1</td>
+<td>async</td>
+<td>Connected to flash (<strong>VCC domain</strong>). Used for flash power management.</td>
+</tr>
+<tr class="even">
+<td>flash_power_ready_h_o</td>
+<td>O</td>
+<td>1</td>
+<td>async</td>
+<td>Connected to flash (<strong>VCC domain</strong>). Used for flash power management.</td>
+</tr>
+<tr class="odd">
+<td>vcmain_pok_o</td>
+<td>O</td>
+<td>1</td>
+<td>async</td>
+<td>Main core power-exist indication. Used by the OpenTitan power manager to determine the state of the main digital supply during power up and power down sequencing.</td>
+</tr>
+<tr class="even">
+<td>vcaon_pok_o</td>
+<td>O</td>
+<td>1</td>
+<td>async</td>
+<td>Always-on power-exist indication. Used by the OpenTitan power manager for power-on reset root.</td>
+</tr>
+<tr class="odd">
+<td>vioa_pok_o</td>
+<td>O</td>
+<td>1</td>
+<td>async</td>
+<td>VIOA power-exist indications. Used as a power-OK status signal.</td>
+</tr>
+<tr class="even">
+<td>viob_pok_o</td>
+<td>O</td>
+<td>1</td>
+<td>async</td>
+<td>VIOB power-exist indication. Used as a power-OK status signal.</td>
+</tr>
+<tr class="odd">
+<td>por_ni</td>
+<td>I</td>
+<td>1</td>
+<td>async</td>
+<td>Power on reset input signal to AST. See <a href="#resets"><u>Resets</u></a> section for further details</td>
+</tr>
+<tr class="even">
+<td>main_pd_ni</td>
+<td>I</td>
+<td>1</td>
+<td>aon</td>
+<td>Power down enable for main core power<br />
+0: main core power is down (deep-sleep state)<br />
+1: main core power is up<br />
+It may take up to 200 uS from this signal transition to power switching completion by AST (not including boot time and so). Note that flash must be prepared for power down before this signal is asserted.</td>
+</tr>
+<tr class="odd">
+<td>main_iso_en_i</td>
+<td>I</td>
+<td>1</td>
+<td>aon</td>
+<td>Isolation enable for main core power (VCMAIN). This signal is fed to AST for any local power clamping needs. This signal is only valid after aon core is available, until then, VCC / AVCC components must take appropriate steps to protect themselves.</td>
+</tr>
+<tr class="even">
+<td><strong>Clock Outputs</strong></td>
+<td></td>
+<td></td>
+<td></td>
+<td></td>
+</tr>
+<tr class="odd">
+<td>clk_src_sys_o</td>
+<td>O</td>
+<td>1</td>
+<td>sys</td>
+<td>100 MHz clock with jitter (main clock domain). Used as the main system clock.</td>
+</tr>
+<tr class="even">
+<td>clk_src_sys_val_o</td>
+<td>O</td>
+<td>1</td>
+<td>async</td>
+<td>System clock valid. Used as "ack" signals for the <a href="https://github.com/lowRISC/opentitan/blob/master/hw/ip/pwrmgr/rtl/pwrmgr_slow_fsm.sv"><u>power manager slow fsm</u></a>.</td>

```

Comment:
@zi-v 
can you remove the link for now? I am updating the power manager documentation, and once that is in, I will circle back and update the link.

Reply:
OK.
@tjaychen, please make such updates in the AST doc source (I plan to using auto conversion for the next PR).